packages/dev/core/src/Engines/nativeEngine.ts

```diff
@@ -2704,4 +2717,18 @@ export class NativeEngine extends Engine {
                 return buffer;
             });
     }
+
+    override startTimeQuery(): Nullable<_TimeToken> {
+        if (!this._gpuFrameTimeToken) {
+            this._gpuFrameTimeToken = new _TimeToken();

```

Comment:
Is there a way to prevent GC ???

Reply:
This object never gets cleared, so there is just one instance created the first time this function is called, and then it uses the same instance indefinitely.